### PR TITLE
Updated to version 2.9 and complete unattended installation

### PR DESCRIPTION
**Pull request by rhessing**
Changes:
* Updated to Spacewalk version 2.9
* Updated the iptables role
* Admin account is automatically created
* To enable a fully automated installation the password question has been removed.
* Default admin password is admin
* Default company name is Spacewalk
* The default information can be changed in this file: ./roles/spacewalk-customisations/defaults/main.yml
* Added support for Ubuntu Bionic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
+Hardware / Virtual sizing advisement:
+- 1 CPU, 4GB of ram minimum, 64GB of disk 
+
+
+
+For the lazy people: 
+To install both ansible and the spacewalk server:
 ```
-sh <(curl -s https://raw.githubusercontent.com/fayaaz/spacewalk.ansible/master/init.sh)
+sh <(curl -s https://raw.githubusercontent.com/rhessing/spacewalk.ansible/master/init-server.sh)
```
Comment (fayaaz) [inline README.md:24]: It would be nice if this pointed to this repo - but can be done in a separate PR

File: init-client.sh [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+#TODO.md
+
+# Always check if a clean install can be updated
+yum -y update
+
+if [ "$1" = "-v" ]; then
+  ANSIBLE_VERSION="${2}"
+fi
+
+yum_makecache_retry() {
```
Comment (fayaaz) [inline init-client.sh:11]: Does this need to be repeated in both scripts?

**Review by fayaaz — APPROVED** [review]

File: init-client.sh (hunk 2 above)
Comment (rhessing) [reply init-client.sh:11]: Well, it could be changed to one script by using simple parameters actually :-)

File: README.md (hunk 1 above)
Comment (rhessing) [reply README.md:24]: yes this has to be done on a view places actually. Lets do that in a seperate PR

**Comment by fayaaz** [comment]
Thanks!

**Pull request merged**